kfs-core/src/main/java/edu/arizona/kfs/fp/document/web/struts/YearEndGeneralErrorCorrectionForm.java

```diff
@@ -0,0 +1,14 @@
+package edu.arizona.kfs.fp.document.web.struts;
+
+public class YearEndGeneralErrorCorrectionForm extends GeneralErrorCorrectionForm {
+
+	public YearEndGeneralErrorCorrectionForm() {
+		super();
+	}
+	
+	@Override
+	protected String getDefaultDocumentTypeName() {
+		return "YEGE";
```

Comment:
Magic string can be pulled from KFSConstants.FinancialDocumentTypeCodes.YEAR_END_GENERAL_ERROR_CORRECTION.